docs/observability/README.md

```diff
@@ -0,0 +1,81 @@
+# Observability
+This README describes observability characteristics of the Fleet Manager
+
+## Metrics
+The file [metrics.go](../../pkg/metrics/metrics.go) creates Prometheus metrics
+of differing types. These are the metrics which are then reported and visualised
+in each Grafana dashboard.
+See [here](https://prometheus.io/docs/concepts/metric_types/) for more info
+about Prometheus metric types.
+
+These metrics are grouped by metric subject: data plane clusters,
+service ('kafka') and reconcilers.
+
+There is also a section of metrics regarding the Observatorium service. See [kas-fleet-manager metrics configmap](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/observability/grafana/grafana-dashboard-kas-fleet-manager-metrics.configmap.yaml#L5460-7247)
```

Comment:
are the links still valid?
we moved out of app-interface IIRC

Reply:
That's correct. I removed this part as it does not apply anymore